applications/vane-network.md

```diff
@@ -0,0 +1,317 @@
+| 2. | Pallet Transaction Claim             |This pallet will be used to resolve stuck transaction based on the delivery tracking data stored.<br>The resolving includes, claiming back the funds and finishing the transaction (sending to beneficiary).<br><br>Mechanism of resolving is handled on OnInitialize block hook. The claimer submits a structured struct claim with neccessary claims and intention. The resolving function will check the truth of the claim by asserting on the specific transaction desired outcome.<br> |
+| 3. | Backend  (Nest JS)                   | All chain ( vane, polkadot/rococo, assethub) api interaction implementation, API chart workflow |
+| 4. | FrontEnd (Next JS)                   |  A front end prototype in Figma, A front end interaction for implemented feature hosted in vercel which will be only for wide screen ( no responsiveness ) as later on (not this grant ) we will implement a mobile application |
+
+
+
+
+
+
+### Milestone 2 - Pallet contract & chain extensions for extending vane-product services.
+
+### Notion templates marketplace as a POC
+
+
+- **Estimated duration:** 1.2 month
+- **FTE:**  2.65
+- **Costs:** 11,000 USD
+
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Video tutorial | We will publish an workshop on X/Twitter that explains [...] (what was done/achieved as part of the grant and how the product should be used) |
+| 1. | pallet transaction claim resolver | We will create a Substrate module that will handle failed transaction based on XCM delivery issues. Allowing users to claim their incomplete transactions|
+| 2. | permissioned pallet contract & chain extensions | A permissioned wasm contract pallet and chain exensions for vane risk free transfer system to be used to extend Vane product services. Account who intend to extend vane product services must apply to be whitelisted to deploy the contract.<br><br>**Chain extensions**<br> This will be limited to vane specific pallets extrinsics only, which includes, pallet_vane_xcm_transfer, pallet_vane_relay_asset, pallet_transfer_claim |
```

Comment:
> Account who intend to extend vane product services must apply to be whitelisted to deploy the contract.

Do you refer to smart contracts here or to some kind of financial contract? If smart contract, does that mean each user has to deploy a smart contract in order to use vane?

Reply:
No user can extend vane functionalities by deploying a smart contract for an application that uses vane chain extensions and base layer functionality ( risk free and trustless transfers)